### PR TITLE
add deno_mysql、deno-checksum、bytes_formater

### PR DESCRIPTION
**Pull request by manyuanrong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,17 +35,20 @@
...
 - [csv](https://github.com/hashrock/deno-fnparse/blob/master/parsers/csv.ts) - A simple CSV parser.
 - [dcc](https://github.com/BoltDoggy/deno#dcc) - Deno Cache Clean, reloading deps when next running.
 - [dejs](https://github.com/syumai/dejs) - ejs template engine for deno.
+- [deno-checksum](https://github.com/manyuanrong/deno-checksum) - Sha1、MD5 algorithms for Deno
 - deno-dotenv
     - [pietvanzoen/deno-dotenv](https://github.com/pietvanzoen/deno-dotenv) - Dotenv handling for deno.
     - [cardosomarcos/deno-dotenv](https://github.com/cardosomarcos/deno-dotenv) - Loads environment variables from .env for deno projects.
 - [deno-fnparse](https://github.com/hashrock/deno-fnparse) - An extremely simple parser combinator for JavaScript.
 - [deno-globrex](https://github.com/hayd/deno-globrex) - Port of globrex to deno, glob to regular expression.
+- [deno-mysql](https://github.com/manyuanrong/deno_mysql) - MySQL database driver for Deno.
```
Comment (kevinkassimo) [inline README.md]: Since this package is marked as WIP, I'll temporarily hold off adding this entry in this PR. The other 2 entries look solid

Comment (manyuanrong) [reply README.md]: It can work now, it will never be perfect, but it can get better later.

Comment (ry) [reply README.md]: I think it should be merged. Many things are not complete - but we have to start somewhere : )

also i've approved https://deno.land/x/mysql/mod.ts

Comment (kevinkassimo) [reply README.md]: @ry Sure.
Slightly tweaked the description to comply with the WIP awesome contributing guide

**Pull request merged**